flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -89,21 +89,35 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                                     + "seconds suffix, daily expression's formation is startTime-endTime, such as 9:30:30-10:50:20, when exclude from 9:30:30-10:50:20 in Monday and Thursday "
                                     + "we can express it as 9:30:30-10:50:20 && * * * ? * 2,5");
 
-    public static final ConfigOption<Double> TARGET_UTILIZATION =
-            autoScalerConfig("target.utilization")
+    public static final ConfigOption<Double> UTILIZATION_TARGET =
+            autoScalerConfig("utilization.target")
                     .doubleType()
                     .defaultValue(0.7)
                     .withFallbackKeys(oldOperatorConfigKey("target.utilization"))
```

Comment:
The new option key is ` job.autoscaler.utilization.target`, but it should be compatible with the following options:

- job.autoscaler.target.utilization (Deprecated key)
- kubernetes.operator.job.autoscaler.target.utilization (fallback key)
- kubernetes.operator.job.autoscaler.utilization.target (fallback key)